src/DropdownMenu.tsx

```diff
@@ -44,6 +70,14 @@ const propTypes = {
   /** Have the dropdown switch to it's opposite placement when necessary to stay on screen. */
   flip: PropTypes.bool,
 
+  /**
+   * Aligns the dropdown menu responsively depending on the breakpoint.
+   * The alignment direction will affect the specified breakpoint or larger.
+   *
+   * *Note: Using responsive alignment will disable Popper usage for positioning.*
+   */
+  align: alignPropTypes,
```

Comment:
does it make sense to support specifying _just_ `'left'` or `'right'` to have this take effect at all sizes?

Reply:
e.g. can we merge with and/or deprecate `alignRight`? right now there's e.g. no easy way to force left- (resp. right-) align at all sizes and disable popper

Reply:
To clarify, you mean something like this?
```
align?: 'left' | 'right' | ResponsiveAlignProp
```
Where `left` and `right` has the ability to use/not use popper and any responsive align = immediately disable popper?

I guess we can add a `usePopper` prop to the dropdown menu too.  Knock out #5186 at the same time.

Reply:
yeah... something like that? but maybe we can make `usePopper` just default to `true` unconditionally? or should we default it to false if `align` is specified?

Reply:
I think we still have to do it conditionally due to responsive alignment classes not working with popper.  `left` and `right` should probably follow the default behavior where popper is enabled?

So `true` for `align={left|right}` and `false` for `align={ResponsiveAlignProp}`?

Reply:
ah, okay, so you can't enable popper at all in those cases? that sounds reasonable i guess. we'd just need to be careful to document it explicitly so it's not confusing. and then this would be less a matter of `usePopper` so much as "disable popper if this is set"

Reply:
Thanks for the feedback.  I checked in the changes.